ims/database/image.py

```diff
@@ -141,8 +141,11 @@ def fetch_id_with_name_from_project(self, name, project_name):
             image = self.connection.session.query(Image). \
                 filter(Image.project.has(name=project_name)).filter_by(
                 name=name).one_or_none()
-            if image is not None:
-                return image.id
+
```

Comment:
This is not really a clean up. Without the changes that I am making here, the db is left in an inconsistent state. I would like to keep this PR short because if I add more stuff (dosctrings, and restructuring) that would delay it's approval. I'll do the cleanup in another PR.